staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -31,6 +31,14 @@ const (
 
 	// SchedulerDefaultProviderName defines the default provider names
 	SchedulerDefaultProviderName = "DefaultProvider"
+
+	// SchedulerDefaultPodInitialBackoffDurationSeconds is the default value for the initial backoff duration
+	// for unschedulable pods.
+	SchedulerDefaultPodInitialBackoffDurationSeconds int64 = 1
+
+	// SchedulerDefaultPodMaxBackoffDurationSeconds is the default value for the max backoff duration
+	// for unschedulable pods.
+	SchedulerDefaultPodMaxBackoffDurationSeconds int64 = 10
```

Comment:
is there a concern that we are making those constants part of the API? @Huang-Wei what do you think?

Reply:
@draveness I am concerned about committing to having those constants as part of the API, I see we already defined few here in the past, but I don't see this pattern in other components (types.go file usually just contain the component config struct), I think this is because defaults.go is supposed to be the only module handling defaults.

If we don't have a place for those constants that doesn't expose them as part of the API, I suggest we remove them, use explicit values as you did before and just add a comment for defaultPriorityQueueOptions saying "To change the default podMaxBackoffDuration used by the scheduler, update the ComponentConfig value in defaults.go, not here.", what do you think?

Reply:
> If we don't have a place for those constants that doesn't expose them as part of the API, I suggest we remove them

 +1 on removing this, I don't feel good to have them here either. 

> use explicit values as you did before and just add a comment for defaultPriorityQueueOptions saying "To change the default podMaxBackoffDuration used by the scheduler, update the ComponentConfig value in defaults.go, not here.", what do you think?

SG